app/commands/codemod.go

```diff
@@ -0,0 +1,240 @@
+
+	"github.com/codelingo/lingo/app/commands/codemod"
+	"github.com/codelingo/lingo/app/util"
+
+	"os"
+
+	"context"
+
+	"github.com/codegangsta/cli"
+	"github.com/codelingo/lingo/app/util/common/config"
+)
+
+var codemodCommand = cli.Command{
+	Name:        "codemod",
+	Usage:       "Modify code following tenets in .lingo.",
+	Subcommands: cli.Commands{*pullRequestCmd},
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  util.LingoFile.String(),
+			Usage: "A .lingo file to perform the review with. If the flag is not set, .lingo files are read from the branch being reviewed.",
+		},
+		cli.StringFlag{
+			Name:  util.DiffFlg.String(),
+			Usage: "Review only unstaged changes in the working tree.",
+		},
+		cli.StringFlag{
+			Name:  util.OutputFlg.String(),
+			Usage: "File to save found issues to.",
+		},
+		cli.StringFlag{
+			Name:  util.FormatFlg.String(),
+			Value: "json-pretty",
+			Usage: "How to format the found issues. Possible values are: json, json-pretty.",
+		},
+		cli.BoolFlag{
+			Name:  util.KeepAllFlg.String(),
+			Usage: "Keep all issues and don't be prompted to confirm each issue.",
+		},
+		cli.StringFlag{
+			Name:  util.DirectoryFlg.String(),
+			Usage: "Modify a given directory.",
+		},
+		cli.BoolFlag{
+			Name:  "debug",
+			Usage: "Display debug messages",
+		},
+		// cli.BoolFlag{
+		// 	Name:  "all",
+		// 	Usage: "review all files under all directories from pwd down",
+		// },
+	},
+	Description: `
+"$ lingo review" will review all code from pwd down.
+"$ lingo review <filename>" will only review named file.
+`[1:],
+	// "$ lingo review" will review any unstaged changes from pwd down.
+	// "$ lingo review [<filename>]" will review any unstaged changes in the named files.
+	// "$ lingo review --all [<filename>]" will review all code in the named files.
+	Action: codemodAction,
+}
+
+func codemodAction(ctx *cli.Context) {
+	err := codemodRequire()
+	if err != nil {
+		util.FatalOSErr(err)
+		return
+	}
+
+	msg, err := codemodCMD(ctx)
+	if err != nil {
+		if ctx.IsSet("debug") {
+			// Debugging
+			util.Logger.Debugw("codemodAction", "err_stack", errors.ErrorStack(err))
+		}
+		util.FatalOSErr(err)
+		return
+	}
+
+	fmt.Println(msg)
+}
+
+func codemodRequire() error {
+	reqs := []require{vcsRq, homeRq, authRq, configRq, versionRq}
+	for _, req := range reqs {
+		err := req.Verify()
+		if err != nil {
+			return errors.Trace(err)
+		}
+	}
+	return nil
+}
+
+func codemodCMD(cliCtx *cli.Context) (string, error) {
+	defer util.Logger.Sync()
+	if cliCtx.IsSet("debug") {
+		util.Logger.Debugw("codemodCMD called")
+	}
+	dir := cliCtx.String("directory")
+	if dir != "" {
+		if err := os.Chdir(dir); err != nil {
+			return "", errors.Trace(err)
+		}
+	}
+
+	dotlingo, err := codemod.
+		ReadDotLingo(cliCtx)
+	if err != nil {
+		return "", errors.Trace(err)
+	}
+	vcsType, repo, err := vcs.New()
+	if err != nil {
+		return "", errors.Trace(err)
+	}
+
+	// TODO: replace this system with nfs-like communication.
```

Comment:
Make sure you add this TODO as an issue, milestone, or in your project management tool; eg. Trello.